src/utils/createEl/createEl.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @function createEl
+ * @desc creates an element from an object
+ * @param {object} attributes for element
+ * @param {string} element the type of element to create
+ * @param {string} className a list of classes to be applied
+ * @param {object} attributes any other attributes passed in.  Must be provided in camelcase
+ * @return {node} - Element with all applied attributes.
+ */
+
+const createEl = ({
+        element = 'div',
+        className,
+        ...attributes
```

Comment:
Nit pickie thing but I actually like having all the params on one line unless it exceeds 100 char width then one per line. Prob need some linting on that. Also the indentation is off.